catalog/app/utils/AWS/Signer.js

```diff
@@ -10,24 +11,48 @@ import {
 } from 'utils/reactTools';
 
 import * as Config from './Config';
+import * as S3 from './S3';
 
 
+const DEFAULT_URL_EXPIRATION = 5 * 60; // in seconds
```

Comment:
why isn't this imported from `/constants`?

Reply:
because it's a default value. if we want to configure this, we should add a config value and then use it when instantiating the provider in `app.js`